src/main/resources/templates/pps/confirmationPage.html

```diff
@@ -15,57 +15,64 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <div class="govuk-panel govuk-panel--confirmation" id="confirmation-details">
-
-                    <div class="govuk-panel__body govuk-!-margin-bottom-6" id="confirmation-company-name-number">
+                    <div class="govuk-panel__body" id="confirmation-company-name-number">
```

Comment:
Maybe we should rename this id, now that we don't have company name / number here?
Should it be something like `id="confirmation-penalty-reference"`